src/js/components/sidebar.jsx

```diff
@@ -53,36 +53,38 @@ class Sidebar extends React.Component {
 
     return (
       <Wrapper clicked={clicked}>
-        <div className="row">
-          <Time
-            opening={opening}
-            closing={closing}
-            id={RestaurantID}
-            handleClick={this.handleClick}
-            clicked={clicked}
-            animate={animate}
-          />
-        </div>
-        <div className="row">
-          <Address
-            restaurantName={restaurantName}
-            streetAddress={streetAddress}
-            city={city}
-            state={state}
-            zipCode={zipCode}
-          />
-        </div>
-        <div className="row">
-          <Phone telephoneNumber={telephoneNumber} />
-        </div>
-        <div className="row">
-          <Website website={website} />
-        </div>
-        <div className="row">
-          <Directions streetAddress={streetAddress} city={city} state={state} zipCode={zipCode} />
-        </div>
-        <div className="row lastRow">
-          <Map streetAddress={streetAddress} city={city} state={state} zipCode={zipCode} />
+        <div id="inner">
```

Comment:
If I understand this correctly, this allows for the right column to have a bigger height than the description box and to be sticky on the page. This is great! Though, considering the way we're assembling our services together this creates a lot of whitespace between your description box and Meredith's module. Just food for thoughts.